README.md

```diff
@@ -362,91 +371,51 @@ Downloading cf-standard-names table version 35 from: http://cfconventions.org/Da
 
 ## Installation
 
-### Non-Python dependencies
+### Conda users
 
-A number of the Python modules used by the compliance checker such as NumPy
-require linking against external C or Fortran libraries.  If using an
- installation method for Python dependencies such as pip below, these external
- libraries will need to be installed beforehand prior to the installation of
- the Python dependencies.
+`compliance-checker` depends on many external C libraries,
+so the easiest way to install it on MS-Windows/OS X/Linux is with `conda`.
+
+```shell
+$ conda install -c conda-forge compliance-checker
+```
+
+For more information on `conda` and installing the IOOS software stack see:
+
+https://github.com/ioos/notebooks_demos/wiki/Installing-Conda-Python-with-the-IOOS-environment
+
+### Pip users
+
+When using pip the user must install the non-Python dependencies first.
 Known C library dependencies include:
-  - NetCDF4
-  - HDF5
   - UDUNITS (2.x series)
+  - HDF5
+  - NetCDF4
   - libxml2/libxslt
 
-Installation for these libraries will vary depending on your choice of
-operating system and method of installation (i.e. binary packages versus
-compiling from source).  For more information on installing these libraries,
+Installation for these libraries will vary depending on your choice of operating system and method of installation
+(i.e. binary packages versus compiling from source).
+For more information on installing these libraries,
 reference the documentation from the individual libraries.
-
-### Python dependencies with `pip`
+(Check our [Ubuntu Install Guide](doc/ubuntu-install-guide.md) out.)
 
 To install locally, set up a virtual environment (recommend using
 [virtualenv-burrito](https://github.com/brainsik/virtualenv-burrito);
-however, do not use `virtualenv-burrito` if you are using Anaconda python.
-[Instructions for Anaconda](#anaconda) appear below):
 
 ```
 $ mkvirtualenv --no-site-packages compliance-checker
 $ workon compliance-checker
 ```
 
-The Python dependencies require several underlying system packages that most package managers should have. See the [Installation](//github.com/ioos/compliance-checker/wiki/Installation) wiki page for more information.
+The Python dependencies require several underlying system packages that most package managers should have.
+See the [Installation](//github.com/ioos/compliance-checker/wiki/Installation) wiki page for more information.
 
 Install dependencies, numpy must be installed on its own:
 
-```
-$ pip install numpy
+```shell
 $ pip install compliance-checker
 ```
 
-###<a name="anaconda"></a> Anaconda on linux
```

Comment:
"Folded" into the instructions above. Although maybe we should bring back the testing part. Not sure b/c the README is already too long.